airflow-core/src/airflow/ui/src/theme.ts

```diff
@@ -34,140 +34,349 @@ const customConfig = defineConfig({
-        // lime
-        running: {
-          "50": { value: "#EFFBEF" },
-          "100": { value: "#DEF7DE" },
-          "200": { value: "#B9EEB9" },
-          "300": { value: "#98E698" },
-          "400": { value: "#78DE78" },
-          "500": { value: "#53D553" },
-          "600": { value: "#32CD32" },
-          "700": { value: "#269C26" },
-          "800": { value: "#196719" },
-          "900": { value: "#0D350D" },
-          "950": { value: "#061906" },
-        },
-        // violet
-        restarting: {
-          "50": { value: "#F6EBFF" },
-          "100": { value: "#EDD6FF" },
-          "200": { value: "#D9A8FF" },
-          "300": { value: "#C880FF" },
-          "400": { value: "#B657FF" },
-          "500": { value: "#A229FF" },
-          "600": { value: "#8F00FF" },
-          "700": { value: "#6E00C2" },
-          "800": { value: "#480080" },
-          "900": { value: "#260042" },
-          "950": { value: "#11001F" },
-        },
-        // mediumpurple
-        deferred: {
-          "50": { value: "#F6F3FC" },
-          "100": { value: "#EDE7F9" },
-          "200": { value: "#DACEF3" },
-          "300": { value: "#C8B6ED" },
-          "400": { value: "#B9A1E7" },
-          "500": { value: "#A689E1" },
-          "600": { value: "#9370DB" },
-          "700": { value: "#6432C8" },
-          "800": { value: "#412182" },
-          "900": { value: "#211041" },
-          "950": { value: "#100821" },
-        },
-        // tan
-        scheduled: {
-          "50": { value: "#FBF8F4" },
-          "100": { value: "#F8F3ED" },
-          "200": { value: "#F1E7DA" },
-          "300": { value: "#E8D9C4" },
-          "400": { value: "#E1CDB2" },
-          "500": { value: "#DAC1A0" },
-          "600": { value: "#D2B48C" },
-          "700": { value: "#B9894B" },
-          "800": { value: "#7D5C31" },
-          "900": { value: "#3E2E18" },
-          "950": { value: "#21180D" },
-        },
-        // lightblue
-        none: {
-          "50": { value: "#F7FBFD" },
-          "100": { value: "#F3F9FB" },
-          "200": { value: "#E4F2F7" },
-          "300": { value: "#D8ECF3" },
-          "400": { value: "#C8E5EE" },
-          "500": { value: "#BDDFEB" },
-          "600": { value: "#ADD8E6" },
-          "700": { value: "#5FB2CE" },
-          "800": { value: "#30819C" },
-          "900": { value: "#18414E" },
-          "950": { value: "#0C2027" },
-        },
-        // lightgrey
-        removed: {
-          "50": { value: "#FCFCFC" },
-          "100": { value: "#F7F7F7" },
-          "200": { value: "#F0F0F0" },
-          "300": { value: "#E8E8E8" },
-          "400": { value: "#E0E0E0" },
-          "500": { value: "#DBDBDB" },
-          "600": { value: "#D3D3D3" },
-          "700": { value: "#9E9E9E" },
-          "800": { value: "#696969" },
-          "900": { value: "#363636" },
-          "950": { value: "#1A1A1A" },
+        black: { value: "#1C202A"},
```

Comment:
@bbovenzi Do you think I need all 11 shades of black to comply with Chakra? It seems like the bg, fg, border semantic tokens should work with just "black" and "white".
https://chakra-ui.com/docs/theming/colors